packages/djangoflow_app/lib/src/presentation/builders/app_builder.dart

```diff
@@ -6,10 +6,10 @@ typedef AppBuilderStateCallBack = Function(BuildContext context);
 class AppBuilder extends StatelessWidget {
   const AppBuilder({
     super.key,
-    required this.providers,
-    this.listeners,
+    this.providers = const [],
```

Comment:
Why did you change the default behavior? Could you please explain your thoughts behind this?